test-fixtures/publish-extra-package-core-dependency.txt

```diff
@@ -0,0 +1,18 @@
+Reading Spago workspace configuration...
+
+✓ Selecting package to build: aaa
+
+Downloading dependencies...
+Building...
+           Src   Lib   All
+Warnings     0     0     0
+Errors       0     0     0
+
+✓ Build succeeded.
+
+Your package "aaa" is not ready for publishing yet, encountered 1 error:
+
+
+✘ Some of the packages you specified as `extra_packages` do not point to the Registry.
+To be able to publish a package to the registry, all of its dependencies have to be packages registered on the Registry.
+Please replace the following packages with versions that are present in the Registry:- "maybe"
```

Comment:
I guess we need newlines for this list of package names

Reply:
@fsoikin we should adjust this error then we are good to merge

Reply:
Done. Sorry, I didn't realize that you implied that I should do it here. Looked like an idle musing kind of statement.

Reply:
Ah yeah, apologies for not being clearer!